java-checks/src/main/java/org/sonar/java/checks/ConstantMethodCheck.java

```diff
@@ -62,6 +65,15 @@ public void visitNode(Tree tree) {
     }
   }
 
+  private static boolean hasDefaultKeyword(MethodTreeImpl methodTree) {
+    Symbol.MethodSymbol symbol = methodTree.symbol();
```

Comment:
Alright, was trying to find some method like that but I couldn't, did the change, thanks!